src/server/services/bezwaren/bezwaren.ts

```diff
@@ -0,0 +1,383 @@
+
+  if (!gevondenKenmerk) {
+    return null;
+  }
+
+  return gevondenKenmerk.bron;
+}
+
+function transformBezwarenResults(
+  response: BezwarenSourceResponse<BezwaarSourceData>
+): Bezwaar[] {
+  const results = response.results;
+  if (Array.isArray(results)) {
+    return results.map((bezwaarBron) => {
+      const besluitdatum = getKenmerkValue(
+        bezwaarBron.kenmerken,
+        'besluitdatum'
+      );
+
+      const bezwaar: Bezwaar = {
+        identificatie: bezwaarBron.identificatie,
+        uuid: bezwaarBron.uuid,
+        ontvangstdatum: bezwaarBron.startdatum,
+        bezwaarnummer: bezwaarBron.identificatie,
+        omschrijving: bezwaarBron.omschrijving,
+        toelichting: bezwaarBron.toelichting,
+        status: getKenmerkValue(bezwaarBron.kenmerken, 'statustekst'),
+        statussen: [],
+        datumbesluit: besluitdatum === null ? null : besluitdatum,
+        datumIntrekking: getKenmerkValue(
+          bezwaarBron.kenmerken,
+          'datumintrekking'
+        ),
+        einddatum: !!bezwaarBron.einddatum ? bezwaarBron.einddatum : null,
+        primairbesluit: getKenmerkValue(bezwaarBron.kenmerken, 'besluitnr'),
+        primairbesluitdatum: getKenmerkValue(
+          bezwaarBron.kenmerken,
+          'besluitdatum'
+        ),
+        resultaat: getKenmerkValue(bezwaarBron.kenmerken, 'resultaattekst'),
+        documenten: [],
+        link: {
+          title: 'Bekijk details',
+          to: generatePath(AppRoutes['BEZWAREN/DETAIL'], {
+            uuid: bezwaarBron.uuid,
+          }),
+        },
+      };
+
+      return bezwaar;
+    });
+  }
+  return [];
+}
+
+function isNotErrorResponse(
+  item: ApiErrorResponse<null> | Bezwaar
+): item is Bezwaar {
+  return item.status !== 'ERROR';
+}
+
+async function enrichBezwaarResponse(
+  bezwarenResponse:
+    | ApiPostponeResponse
+    | ApiErrorResponse<null>
+    | ApiSuccessResponse<Bezwaar[]>,
+  authProfileAndToken: AuthProfileAndToken
+) {
+  if (bezwarenResponse.status !== 'OK') {
+    return [];
+  }
+
+  const enrichtedList = bezwarenResponse.content.map(async (bezwaar) => {
+    const statussen: BezwaarStatus[] = await fetchBezwaarStatus(
+      bezwaar.uuid,
+      authProfileAndToken
+    );
+    const documenten =
+      (await fetchBezwarenDocuments(bezwaar.uuid, authProfileAndToken)) ?? [];
+
+    const enrichedBezwaar: Bezwaar = {
+      ...bezwaar,
+      statussen,
+      documenten,
+    };
+
+    return enrichedBezwaar;
+  });
+
+  const content = await Promise.allSettled(enrichtedList);
+  const results = content.map((res) => getSettledResult(res));
+
+  return results.filter(isNotErrorResponse);
+}
+
+export async function fetchBezwaren(
+  requestID: requestID,
+  authProfileAndToken: AuthProfileAndToken
+): Promise<ApiResponse<Bezwaar[] | null>> {
+  const requestBody = JSON.stringify({
+    [getIdAttribute(authProfileAndToken)]:
+      process.env.BFF_BEZWAREN_TEST_BSN ?? authProfileAndToken.profile.id,
```

Comment:
```suggestion
    [getIdAttribute(authProfileAndToken)]: authProfileAndToken.profile.id,
```

Ik heb dit al eerder ingebouwd voor Yivi op een ander niveau: `BFF_PROFILE_DEV_ID` dit zorgt ervoor dat je direct al in de loginflow/sessie het juiste id (bsn, kvk, email) te pakken krijgt.